dou_downloads/1993/DO3/busca.py

```
@@ -0,0 +1,158 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import argparse
+import csv
+import os
+import re
+import sys
+import unicodedata
+import zipfile
+from datetime import datetime
```

Ruff (0.12.2)

`datetime.datetime` imported but unused

Remove unused import: `datetime.datetime`

(F401)